diagnostic.js

```diff
@@ -41,7 +46,7 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.remove({ motto: "As High as Honor"})
```

Comment:
Not quite. This is `remove` gets rid of the entire document from the collection. This is actually an update:

```js
db.houses.update({ name: "House Arryn"}, { name: "House Arryn"});
```